scripts/system-stats.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+#
+# Reports cpu and ram usage statistics
+#
+set -e
+
+[[ $(uname) == Linux ]] || exit 0
+
+# need to cd like this to avoid #SC1091
+cd "$(dirname "$0")/.."
+source scripts/configure-metrics.sh
+cd scripts
```

Comment:
@mvines I had to choose between relative-path script invocation in line 30 or just cd back into the scripts dir here. Does this look ok?

Reply:
I'd just remove `cd scripts` entirely and reference `scripts/metrics-write-datapoint.sh`